tests/playwright-test/reporter-raw.spec.ts

```diff
@@ -238,6 +238,24 @@ test(`testInfo.attach should save attachments via inline attachment`, async ({ r
   }
 });
 
+test(`testInfo.attach should handle empty body attachments`, async ({ runInlineTest }, testInfo) => {
+  await runInlineTest({
+    'a.test.js': `
+      const path = require('path');
+      const fs = require('fs');
+      const { test } = pwt;
+      test('empty', async ({}, testInfo) => {
+        await testInfo.attach('example.txt', { body: '' });
+      });
+  `,
+  }, { reporter: 'dot,' + kRawReporterPath, workers: 1 }, {}, { usesCustomOutputDir: true });
+  const json = JSON.parse(fs.readFileSync(testInfo.outputPath('test-results', 'report', 'project.report'), 'utf-8'));
+  const result = json.suites[0].tests[0].results[0];
+  expect(result.attachments[0].name).toBe('example.txt');
+  expect(result.attachments[0].contentType).toBe('text/plain');
+  expect(Buffer.from(result.attachments[0].body, 'base64')).toEqual(Buffer.from(''));
+});
+
```

Comment:
```suggestion
test(`testInfo.attach should handle empty body attachments`, async ({ runInlineTest }, testInfo) => {
  await runInlineTest({
    'a.test.js': `
      const path = require('path');
      const fs = require('fs');
      const { test } = pwt;
      test('empty', async ({}, testInfo) => {
        await testInfo.attach('example.txt', { body: '' });
      });
  `,
  }, { reporter: 'dot,' + kRawReporterPath, workers: 1 }, {}, { usesCustomOutputDir: true });
  const json = JSON.parse(fs.readFileSync(testInfo.outputPath('test-results', 'report', 'project.report'), 'utf-8'));
  const result = json.suites[0].tests[0].results[0];
  expect(result.attachments[0].name).toBe('example.txt');
  expect(result.attachments[0].contentType).toBe('text/plain');
  expect(Buffer.from(result.attachments[0].body, 'base64')).toEqual(Buffer.from(''));
});
```

Unlike the other test cases, this one may be unnecessary since raw reporter sees body's as Buffers, not strings. Let me know if you this case deleted!

Reply:
Yep, I don't think we should duplicate all attachment tests for raw reporter as well 😄

Reply:
👍 Just removed it. (Had to do it on the CLI. First attempt via the GitHub suggestions PR UI just resulted in a single line deletion…guess it doesn't like deleting the whole chunk! hehe)